core/test/audits/prioritize-lcp-image-test.js

```diff
@@ -61,19 +70,21 @@ describe('Performance: prioritize-lcp-image audit', () => {
         isLinkPreload: false,
         networkRequestTime: 500,
         networkEndTime: 1000,
-        timing: {receiveHeadersEnd: 500},
+        transferSize: 16_000,
         url: mainDocumentNodeUrl,
+        frameId: 'ROOT_FRAME',
       },
       {
         requestId: '3',
         resourceType: 'Script',
         priority: 'High',
         isLinkPreload: false,
         networkRequestTime: 1000,
-        networkEndTime: 5000,
```

Comment:
and it didn't make sense for the LCP image initiator to finish after the LCP image request (future script preload scanner? :) but more importantly after the 4500ms LCP. The simulator did its best, but again made changing any tests and getting a different result difficult.